hooks/command

```diff
@@ -343,6 +343,10 @@ elif plugin_read_list_into_result BUILDKITE_PLUGIN_DOCKER_SHELL ; then
   done
 fi
 
+# Add a container name
+container_name="${BUILDKITE_PIPELINE_SLUG}_${BUILDKITE_BUILD_NUMBER}"
```

Comment:
The docker-compose plugin integrates `BUILDKITE_JOB_ID` into the container name, which means the name is unique per job (which includes being different across retries of a failed build step etc):

https://github.com/buildkite-plugins/docker-compose-buildkite-plugin/blob/2f44a9c309336df0558d21b67770f5f89f4002fa/lib/shared.bash#L84

I think we might need to do something similar here?
Otherwise a single agent machine (with a single docker daemon) could concurrently run two jobs in the same build that end up with the same `container_name`.